test/circular/main.js

```diff
@@ -0,0 +1,6 @@
+import { getBar } from './foo';
+import { getFoo } from './bar';
+
+// true in dev, false in production
```

Comment:
Comment was old, when this was breaking it was true in dev and false in prod. It's working now, just forgot to remove the comment.